projects/kit/components/input-number/input-number.component.ts

```diff
@@ -281,6 +269,26 @@ export class TuiInputNumberComponent
         this.updatePressed(pressed);
     }
 
+    getFormattedValue(value: number): string {
```

Comment:
Private?

Reply:
This method is required also inside `InputSlider`